examples/vae/ss_vae_M2.py

```diff
@@ -299,9 +299,6 @@ def main(args):
     :param args: arguments for SS-VAE
     :return: None
     """
-    if args.cuda:
```

Comment:
@jpchen - I removed the global setting here by removing these lines - so, this should prevent pytorch from asking torch.randperm to return a cuda tensor (which was the main error that showed up). but because of this change, we can't use torch.zeros and torch.ones in the model when we want the loss to be computed on GPUs -- hence the other fixes.